modules/insights-operator-understanding-archive.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-operator.adoc
+
+:_content-type: CONCEPT
+[id="insights-operator-understanding-archive_{context}"]
+= Understanding the Insights Operator Archive
+
+Insights Operator stores gathered data in an archive located in the `openshift-insights` namespace of your cluster. Users of {product-title} can inspect this archive at any time. 
+
```

Comment:
@tremes I roughed-in this module a bit and it is still a WIP. Do you think there is any important information regarding the IO archive that we can add? Maybe file structure etc.? If not, we can always remove this one.

Reply:
I think the important info is https://github.com/openshift/openshift-docs/pull/44542/files#diff-0e31965b2bc540504561d767252476e3e9ce964b3630bf8573607c8a3540e2b0R18 and I wouldn't add any file/dir structure TBH.